docs/Local-Dispatcher-Windows.md

```diff
@@ -1 +1,126 @@
-# Running a Dispatcher Locally on Windows
+# Running a Dispatcher Locally on Windows IIS
+
+There are a few different versions of Windows and at least 2 ways to set up Dispatcher on them.  The Apache
+Webserver approach is possible, but these instructions are for using the recommended server: _Internet
+Information Services (IIS)_ and leans towards Windows 10.
+
+The instructions largely follow the
+[AEM Documentation](https://experienceleague.adobe.com/docs/experience-manager-dispatcher/using/getting-started/dispatcher-install.html)
+with additional notes, to help avoid problems.
+
+## Host name "aem-publish.local"
+
+Following the AEM instructions will set up your dispatcher on host "localhost" by default.  To
+follow the dispatcher experiments more closely, the host name "**aem-publish.local**" should be used.
+To do that, follow these instructions which loosely follow some of the steps found in this web page:
+[tutorialsteacher](https://www.tutorialsteacher.com/articles/set-custom-domain-name-for-localhost-website-iis)
+
+- In IIS Manager, right click on the project name (top level on the left-hand tree)
+- Click "Add Website..."
+- Enter:
+  - Site Name: **aem-publish.local**
+  - Physical Path: a path anywhere on your filesystem
+    - maybe create this folder close to your author & publish instances for convenience, or under \inetpub
+    - this is where the cached files will reside
+  - Bind to port: 8080
+  - Host Name: **aem-publish.local**
+- Edit your _hosts_ file
+  - This file is often found at "C:\Windows\System32\drivers\etc\"
+  - Add `127.0.0.1 aem-publish.local` at the end of the file
+- Try "http://aem-publish.local:8080" - this will produce poor results, but should produce a default server
+error response, which proves the site is up and running.
+- To avoid confusion, stop the "Default Web Site" site
+
+Continue with the instructions below, but instead of making changes to the default site, make the changes to the
+new **aem-publish.local** site.  That is your "Dispatcher" or "Dispatcher cache" site now.
+
+## Dispatcher Installation Instructions
+
+Follow the Windows/IIS instructions from the following documentation: 
+[Experience Manager Dispatcher Help - Installing Dispatcher](https://docs.adobe.com/content/help/en/experience-manager-dispatcher/using/getting-started/dispatcher-install.html)
```

Comment:
Looks like this page has been moved to experienceleague.adobe.com: https://experienceleague.adobe.com/docs/experience-manager-dispatcher/using/getting-started/dispatcher-install.html